modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml

```diff
@@ -373,4 +373,9 @@
             <class name="org.wso2.am.integration.tests.workflow.WorkflowApprovalExecutorTest"/>
         </classes>
     </test>
+    <test name="apim-integration-tests-soap" preserve-order="true" parallel="false" group-by-instances="true">
+        <classes>
+            <class name="org.wso2.am.integration.tests.other.WSDLImportTestCase"/>
+        </classes>
+    </test>
```

Comment:
We do not need to add this here. This is already there under,

```<test name="apim-integration-tests-samples" preserve-order="true" parallel="false" group-by-instances="true">```

Reply:
fixed